docs/sections/user_guide/cli/drivers/chgres_cube.rst

```diff
@@ -0,0 +1,98 @@
+``chgres_cube``
+===============
+
+The ``uw`` mode for configuring and running the :ufs-utils:`chgres_cube<chgres-cube>` component.
+
+.. code-block:: text
+
+   $ uw chgres_cube --help
+   usage: uw chgres_cube [-h] TASK ...
+
+   Execute chgres_cube tasks
+
+   Optional arguments:
+     -h, --help
+         Show help and exit
+
+   Positional arguments:
+     TASK
+       namelist_file
+         The namelist file
+       provisioned_run_directory
+         Run directory provisioned with all required content
+       run
+         A run
+       runscript
+         The runscript
+
+
+All tasks take the same arguments. For example:
+
+.. code-block:: text
+
+   $ uw chgres_cube run --help
+   usage: uw chgres_cube run --config-file PATH --cycle CYCLE [-h] [--batch] [--dry-run]
+                             [--graph-file PATH] [--quiet] [--verbose]
+
+   A run
```

Comment:
Change to --
```
      Run execution
```
to be consistent with sfc_climo_gen

Reply:
I think I might go the opposite direction here and update sfc_climo_gen, instead. The fv3 driver has "A run" and I think it's less clunky,